core/src/main/codegen/templates/Parser.jj

```diff
@@ -3422,7 +3429,6 @@ SqlNode AtomicRowExpression() :
 }
 {
     (
-        LOOKAHEAD(1)
```

Comment:
This one is removed cause it is redundant. (Same reason, production `Literal()` contains only a single token)